### PR TITLE
Added Join to Fluent's query page

### PR DESCRIPTION
**Pull request by bygri**

### CODE REVIEW

File: 2.0/docs/fluent/query.md [hunk 1]
```diff
@@ -189,6 +189,29 @@ To sort the results of your query, use the `.sort()` method
...
+Departments contain Employees who have completed ten years of service.
+
+First you use the `.join()` method on a Department query to join it with the Employee table. Next you chain a
+`.filter()` on to the query. Bear in mind you need to explicitly pass the 'joined' model to the filter, otherwise
+Fluent will try to filter on the 'base' model.
+
+```swift
+let departments = try Department.makeQuery()
+  .join(Employee.self)
+  .filter(Employee.self, "years_of_service", .greaterThanOrEquals, 10)
```
Comment (tanner0101) [inline 2.0/docs/fluent/query.md]: Does `.filter(Employee.self, "years_of_service" >= 10)` work here?

Comment (bygri) [inline 2.0/docs/fluent/query.md]: What do you know, it does! I have updated the PR.

**Comment by bygri** [comment]
While I was improving the code snippet in this PR, someone asked on Slack about multi-column sorting, so I added that into the Sort section as well while Github was down.

**Comment by tanner0101** [comment]
Awesome, thanks @bygri!

**Pull request merged**